app/templates/public/index.hbs

```diff
@@ -20,6 +20,7 @@
                                  order=model.order
                                  attendees=model.attendees
                                  code=code
+                                 taxInfo=model.event.tax
```

Comment:
@uds5501 Please modify the test of `Public/Ticket List` to include the `Tax Object`

Reply:
@kushthedude sure, will do it in the next commit